packages/kbn-esql-validation-autocomplete/src/autocomplete/factories.ts

```diff
@@ -30,6 +30,7 @@ const allFunctions = statsAggregationFunctionDefinitions
   .concat(groupingFunctionDefinitions);
 
 export const TIME_SYSTEM_PARAMS = ['?t_start', '?t_end'];
+export const ADD_DATE_HISTOGRAM_SNIPPET = 'BUCKET($0, 20, ?start, ?end)';
```

Comment:
```suggestion
export const ADD_DATE_HISTOGRAM_SNIPPET = 'BUCKET($0, 20, ?t_start, ?t_end)';
```

Also, I have no idea if `20` is the right number here. Not necessarily something we have to block this PR on, but could you make sure we discuss this as a team?